deployment/k8s/sm-secret.yml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  sm-postgre-service-name
```

Comment:
how about replacing the secret with the db name and the secret with the db credentials with one that contains the db uri and then set and env variable in the sm-deployment.yml with name SM_DB_URI and value the val from the secret. And then you don't need  the file cmd/k8s/env/env.go